### PR TITLE
Blog post about hawkinit - simple cli tool for setting up h-services, and some WFs

### PR DESCRIPTION
**Pull request by jkremser**
@pilhuhn || @theute  could you please look?

### CODE REVIEW

File: src/main/jbake/content/blog/2016/12/22/hawkular-hawkinit-cli-tool.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+= Hawkinit
+Jirka Kremser
+2016-12-16
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/12/22/hawkular-hawkinit-cli-tool.adoc]: Use a more current date

File: src/main/jbake/content/blog/2016/12/22/hawkular-hawkinit-cli-tool.adoc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, cli, nodejs
+
+== Introducing Hawkinit
+This simple CLI tool written in NodeJS will help you to set up the running instance of http://localhost:8080/hawkular-services/docs/quickstart-guide/[hawkular-services], one to n instances of Cassandra,
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/12/22/hawkular-hawkinit-cli-tool.adoc]: Hawkular-Services 

one to n -> multiple (?)

File: src/main/jbake/content/blog/2016/12/22/hawkular-hawkinit-cli-tool.adoc [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+and also the instances of WildFly servers that have the hawkular agent installed and configured to report the metrics to the hawkular-services.
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/12/22/hawkular-hawkinit-cli-tool.adoc]: s/hawkular/Hawkular/g

File: src/main/jbake/content/blog/2016/12/22/hawkular-hawkinit-cli-tool.adoc [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+User can select if he wants the WildFly server running in the standalone mode or as a managed domain. She can also choose how many instances it should spawn.
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/12/22/hawkular-hawkinit-cli-tool.adoc:12]: User -> The user (?)

File: src/main/jbake/content/blog/2016/12/22/hawkular-hawkinit-cli-tool.adoc [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+If the domain mode is selected, there are some prepared https://github.com/Jiri-Kremser/hawkfly-domain-dockerfiles#scenarios[scenarios] with different profiles and different amount of servers and server groups per host controller.
+Instead of standalone mode, one can also spawn domain with multiple host controllers simulating the complex real world scenario in couple of seconds.
+
+Under the hood, the application dynamically creates the `docker-compose.yml` file in the temp directory with the parameters obtained as the answers and run the services
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/12/22/hawkular-hawkinit-cli-tool.adoc]: creates the -> creates a  (?)

**Comment by jkremser** [comment]
ouch, https://travis-ci.org/hawkular/hawkular.github.io#L2339

perhaps the gif was too large for the mvn git plugin to push. I am investigating..

**Pull request merged**